docs/examples/ex02.py

```diff
@@ -91,18 +101,12 @@ def C(T):
 K = asm(bilinf, ib)
 f = 1e6 * asm(unit_load, ib)
 
-dofs = ib.find_dofs({
-    'left':  m.facets_satisfying(lambda x: x[0] == 0),
-    'right': m.facets_satisfying(lambda x: x[0] == 1),
-    'top':   m.facets_satisfying(lambda x: x[1] == 1),
-})
-
-D = np.concatenate((
-    dofs['left'].nodal['u'],
-    dofs['left'].facet['u_n'],
-    dofs['right'].nodal['u'],
-    dofs['top'].nodal['u'],
-))
+D = np.concatenate(
+    [
+        ib.get_dofs("left").facet["u_n"],
+        *[ib.get_dofs(s).nodal["u"] for s in ["left", "right", "top"]],
+    ]
+)
```

Comment:
Something like `np.hstack([ib.get_dofs('left'), ib.get_dofs(('right', 'top')).all('u')])` after #778 ?

Reply:
I don't suggest using `dofs1 | dofs2` because I noticed it can lead to unexpected behavior. I suppose it should be deprecated.

Reply:
Is `hstack` going to be the recommended way to concatenate `DofsViews`?

Reply:
Yes, I think so. Using those `|` and `+` operators you may get unintuitive result if `skip` was used in `get_dofs`.

Reply:
Thanks, yes, that's better.